themes/bootstrap3/templates/myresearch/historicloans.phtml

```diff
@@ -0,0 +1,131 @@
+<?
+  // Set up page title:
+  $this->headTitle($this->translate('Loan History'));
+
+  // Set up breadcrumbs:
+  $this->layout()->breadcrumbs = '<li><a href="' . $this->url('myresearch-home') . '">' . $this->transEsc('Your Account') . '</a></li> <li class="active">' . $this->transEsc('Loan History') . '</li>';
+?>
+
+<div class="<?=$this->layoutClass('mainbody')?>">
+  <h2><?=$this->transEsc('Your Loan History')?></h2>
```

Comment:
Do we really need the "Your Loan History" string here and in the language files? Should we just use "Loan History" everywhere for consistency? (There's been conversation in the past about simplifying/reducing "Your" language in the interface... I'm wondering if it's beneficial to avoid adding more of it).

Reply:
Changed. It was just for consistency with the still existing "Your" stuff in checkedout.phtml.